package.json

```diff
@@ -59,6 +59,7 @@
     "@govflanders/vl-ui-icon": "^3.12.24",
     "@govflanders/vl-ui-image": "^3.12.3",
     "@govflanders/vl-ui-infoblock": "^3.12.17",
+    "@govflanders/vl-ui-infotext": "^3.12.3",
```

Comment:
Op deze manier gaan de teams de component niet kunnen gebruiken. in `src/components/infotext/index.js` importeer je `@govflanders/vl-ui-infotext/dist/js/infotext.js` rechtstreeks, dan is dat een dependency en geen devdependency. Op deze manier gaan de teams die package niet binnenhalen en krijgen ze errors.

Reply:
Aangepast naar een dependency.